cmd/res/devices/devices.yaml

```diff
@@ -0,0 +1,88 @@
+# Pre-define Devices
+DeviceList:
```

Comment:
The issue would be the same as this
https://github.com/edgexfoundry/device-onvif-camera/pull/281#discussion_r1147218453

We have to modify the struct yaml tag or this YAML file